kebab/src/raw.ts

```diff
@@ -53,9 +53,9 @@ export const connect = (
   token: Token,
   refreshToken: Token,
   {
-    logger = () => {},
-    onConnectionTaken = () => {},
-    onClearTokens = () => {},
+    logger = () => { },
+    onConnectionTaken = () => { },
+    onClearTokens = () => { },
```

Comment:
Turn off formatting on save in VSCode

Reply:
vro i like auto format. but i dont like this, tell me how to turn only this off

Reply:
VSCode settings.json `"editor.formatOnSave": false`

Reply:
@BronzW i have that off. i format manually. i want to know how to stop it from adding spaces to `{}`